From c21d1791ae7a34ea85bcd9544bf811b944fc2777 Mon Sep 17 00:00:00 2001
From: Roger Leigh <rleigh@debian.org>
Date: Mon, 21 May 2012 20:06:26 +0100
Subject: [PATCH] 10clock-setup: Revert use of /etc/default/hwclock

This was never used for the UTC setting given that we migrate it
directly from /etc/default/rcS, so it's no longer needed it
clock-setup.
---
 debian/changelog               |    7 ++++++-
 finish-install.d/10clock-setup |    6 +-----
 2 files changed, 7 insertions(+), 6 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 3e627c7..3b3912a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -6,7 +6,12 @@ clock-setup (0.111) UNRELEASED; urgency=low
   [ Joey Hess ]
   * Set UTC or LOCAL in /etc/adjtime for systemd. Closes: #660093
 
- -- Samuel Thibault <sthibault@debian.org>  Sun, 12 Feb 2012 23:05:52 +0100
+  [ Roger Leigh ]
+  * Migrate UTC setting from /etc/default/rcS; revert use of
+    /etc/default/hwclock, which is not used for holding the UTC
+    setting.
+
+ -- Roger Leigh <rleigh@debian.org>  Mon, 21 May 2012 20:03:36 +0100
 
 clock-setup (0.110) unstable; urgency=low
 
diff --git a/finish-install.d/10clock-setup b/finish-install.d/10clock-setup
index 4e52af5..42c9a08 100755
--- a/finish-install.d/10clock-setup
+++ b/finish-install.d/10clock-setup
@@ -92,11 +92,7 @@ if ! db_go; then
 fi
 
 # Update target system configuration for utc/localtime selection
-if [ -f /target/etc/default/hwclock ]; then
-	utcfile=/target/etc/default/hwclock
-else
-	utcfile=/target/etc/default/rcS
-fi
+utcfile=/target/etc/default/rcS
 
 db_get clock-setup/utc
 if [ "$RET" = true ]; then
-- 
1.7.10

